### PR TITLE
Cxp 893 add icons

### PR DESCRIPTION
**Pull request by yasskin**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [ ] Edge (Win 10)
- [ ] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

docspot
https://docspot.adnxs.net/projects/lucid/CXP-893-add-icons/?path=/docs/documentation-introduction--introduction

### CODE REVIEW

**Comment by rsadres** [comment]
Hey! Looks like the Hide and Save icons aren't fully rendering. The hide icon is missing the line through the eye, and the save icon isn't showing the outline of the disk. Thanks!
Rachel

**Comment by yasskin** [comment]
> Hey! Looks like the Hide and Save icons aren't fully rendering. The hide icon is missing the line through the eye, and the save icon isn't showing the outline of the disk. Thanks!
> Rachel

Good eye. Please take a look now:

Hide Icon: 
https://docspot.adnxs.net/projects/lucid/CXP-893-add-icons/?path=/docs/icons-icons-hideicon--default
Save Icon:
https://docspot.adnxs.net/projects/lucid/CXP-893-add-icons/?path=/docs/icons-icons-saveicon--default

File: src/components/Icon/GetMaximumIcon/GetMaximumIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import React from 'react';
+import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { FC, omitProps } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-GetMaximumIcon');
+
+interface IGetMaximumIconProps extends IIconProps {}
+
+export const GetMaximumIcon: FC<IGetMaximumIconProps> = ({
```
[the remark this answers is not in the document]
Comment (yasskin) [reply src/components/Icon/GetMaximumIcon/GetMaximumIcon.tsx]: Good catch. Removed. Replaced by the props annotation.

I also added in this line on 43:
`GetMaximumIcon.defaultProps = Icon.defaultProps;`

File: src/components/Icon/HideIcon/HideIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import React from 'react';
+import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { FC, omitProps } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-HideIcon');
+
+interface IHideIconProps extends IIconProps {}
+
+export const HideIcon: FC<IHideIconProps> = ({
```
[the remark this answers is not in the document]
Comment (yasskin) [reply src/components/Icon/HideIcon/HideIcon.tsx]: Ditt. I added this line too on 38:
`HideIcon.defaultProps = Icon.defaultProps;`

File: src/components/Icon/SaveIcon/SaveIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import React from 'react';
+import Icon, { IIconProps, propTypes as iconPropTypes } from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { FC, omitProps } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-SaveIcon');
+
+interface ISaveIconProps extends IIconProps {}
+
+export const SaveIcon: FC<ISaveIconProps> = ({
```
[the remark this answers is not in the document]
Comment (yasskin) [reply src/components/Icon/SaveIcon/SaveIcon.tsx]: Ditto again. With:
`SaveIcon.defaultProps = Icon.defaultProps;`

**Comment by yasskin** [comment]
`BookIcon.defaultProps = Icon.defaultProps;`

Icon’s default props is:
```
const defaultProps = {
    size: 16,
    aspectRatio: 'xMidYMid meet',
    viewBox: '0 0 16 16',
    isDisabled: false,
    isClickable: false,
    color: Color.primary,
    onClick: _.noop,
    onSelect: _.noop,
};
``` 
I set the other Icons the same. Just want to double check that this is correct.
```
SaveIcon.defaultProps = Icon.defaultProps;
GetMaximumIcon.defaultProps = Icon.defaultProps;
HideIcon.defaultProps = Icon.defaultProps;
```

**Comment by jondlm** [comment]
Yeah that looks right to me

File: docs/load-icons.js [hunk 4]
```diff
@@ -567,6 +567,23 @@ module.exports = [
...
 
+	{
+		name: 'GetMaximumIcon',
+		component: getDefaultExport(
+			require('../src/components/Icon/GetMaximumIcon/GetMaximumIcon')
+		),
+		examplesContext: require.context(
+			'../src/components/Icon/GetMaximumIcon/examples',
+			true,
+			/\.jsx?$/
```
Comment (jondlm) [inline docs/load-icons.js]: No need to do it for this PR (it's not super important for such simple components) but new examples should we written in TS moving forward.

Comment (yasskin) [reply docs/load-icons.js]: Sounds good. Will do.

**Pull request merged**